spec/API/book_api_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+describe Book, type: :request do
```

Comment:
Yeah, that was a typo and I didn't commit fix to that yet, but I've done this before performing tests and result is the same.